tests/test_validator.py

```diff
@@ -145,41 +145,21 @@ def test_data_with_undefined_references_to_required_fields_fails_validation(self
         required = ["x", "z"]
 
         test_data = [
-            data(name=name, fields=fields[:i], required=[required[i]]) for i in range(len(fields))
+            data(name=name, fields=fields[:i], required=[required[i]])
+            for i in range(1, len(fields))
         ]
 
         for d in test_data:
             assert_model_is_invalid(self, d, pattern)
 
-    def test_valid_usecases_with_only_participants_pass_validation(self):
```

Comment:
Were these removed because the tests are invalid with empty required lists?

Reply:
Yes